tests/helper/prometheus/prometheus_helper.go

```diff
@@ -0,0 +1,510 @@
+//go:build e2e
+// +build e2e
+
+package helper
```

Comment:
I'd put this prometheus related helper next to the prometheus scaler in the same package and not into this generic helper package.

Reply:
the problem is that it's used for predictkube and also for prometheus, that's why I moved to the shared helpers

Reply:
I mean, I cannot put inside predictkube package because I'll need on prometheus package too

Reply:
:(